betterj.go

```diff
@@ -0,0 +1,35 @@
+package betterj
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+//I used the bytes.buffer because its dynamic and works good with compact and indent
+
+func MinifyJ(j string) (string, error) {
```

Comment:
Add a check what if there is an empty array passed in function as a parameter?

Reply:
Acknowledged